--- lemon-1.2.4/m4/lx_check_glpk.m4.orig	2013-08-10 12:25:53.000000000 +0200
+++ lemon-1.2.4/m4/lx_check_glpk.m4	2013-12-30 22:18:33.502163306 +0100
@@ -49,9 +49,9 @@
 
       int main(int argc, char** argv)
       {
-        LPX *lp;
-        lp = lpx_create_prob();
-        lpx_delete_prob(lp);
+        glp_prob *lp;
+        lp = glp_create_prob();
+        glp_delete_prob(lp);
         return 0;
       }'
 
